src/main/java/seedu/address/logic/parser/ParserType.java

```diff
@@ -1,19 +1,33 @@
 package seedu.address.logic.parser;
 
-import seedu.address.logic.commands.taskcommands.AddTaskCommand;
-import seedu.address.logic.commands.taskcommands.DeleteTaskCommand;
+import java.lang.reflect.Field;
 
-public abstract class ParserType {
+public class ParserType {
+		
+	private static final Class<?>[] parserTypes = new Class[]{AddCommandParser.class, DeleteCommandParser.class};
```

Comment:
One possible improvement is that we move the list of commands to its own static class that only contains this list. That separates the data (`parserTypes`) from the behavior (`get`). This is debatable though - let me know what you think.